newrelic/hooks/framework_graphql.py

```diff
@@ -136,7 +135,14 @@ def wrap_execute_operation(wrapped, instance, args, kwargs):
     if operation.selection_set is not None:
         fields = operation.selection_set.selections
         # Ignore transactions for introspection queries
-        if not settings.instrumentation.graphql.capture_introspection_queries:
+        capture_introspection_queries = (
```

Comment:
Can't this just be left as it was before since the transaction we know must exist now if it got this far? Looking at other places in the code we don't seem to guard against the settings path not existing like this so wondering if this is really needed.
```
if not transaction.settings.instrumentation.graphql.capture_introspection_queries:
```

Reply:
We do sometimes and sometimes not, it's a mixed bag of style across the agent. I'm not sure it's any safer as long as the top level settings exists so maybe we can leave it at that.